test failure

Possibly related to PR #140 

To reproduce:
```
$ cargo build --release
$ cargo test --release
```

Output of `cargo test --release`:
```
...
   Compiling mmb_rpc v0.1.0 (/Users/jgarzik/repo/purefinance/mmb/mmb_rpc)
   Compiling core_tests v0.1.0 (/Users/jgarzik/repo/purefinance/mmb/core_tests)
   Compiling example v0.1.0 (/Users/jgarzik/repo/purefinance/mmb/example)
   Compiling binance v0.1.0 (/Users/jgarzik/repo/purefinance/mmb/exchanges/binance)
   Compiling semver-parser v0.10.2
error[E0599]: no method named `get_specific_currency_pair` found for struct `Arc<mmb_core::core::exchanges::general::exchange::Exchange>` in the current scope
   --> exchanges/binance/tests/binance/binance_builder.rs:126:22
    |
126 |             exchange.get_specific_currency_pair(OrderProxy::default_currency_pair()),
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^ method not found in `Arc<mmb_core::core::exchanges::general::exchange::Exchange>`

error[E0599]: no method named `get_specific_currency_pair` found for struct `dashmap::mapref::one::Ref<'_, mmb_core::core::exchanges::common::ExchangeAccountId, Arc<mmb_core::core::exchanges::general::exchange::Exchange>>` in the current scope
   --> exchanges/binance/tests/control_panel/statistics.rs:134:22
    |
134 |             exchange.get_specific_currency_pair(test_currency_pair),
    |                      ^^^^^^^^^^^^^^^^^^^^^^^^^^ method not found in `dashmap::mapref::one::Ref<'_, mmb_core::core::exchanges::common::ExchangeAccountId, Arc<mmb_core::core::exchanges::general::exchange::Exchange>>`

For more information about this error, try `rustc --explain E0599`.
error: could not compile `binance` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/core/src/core/exchanges/general/exchange.rs
+++ b/core/src/core/exchanges/general/exchange.rs
@@ -17,9 +17,9 @@ use super::commission::Commission;
 use super::polling_timeout_manager::PollingTimeoutManager;
 use super::symbol::Symbol;
 use crate::core::connectivity::connectivity_manager::GetWSParamsCallback;
-#[cfg(debug_assertions)]
-use crate::core::exchanges::common::SpecificCurrencyPair;
-use crate::core::exchanges::common::{ActivePosition, ClosedPosition, TradePlace};
+use crate::core::exchanges::common::{
+    ActivePosition, ClosedPosition, SpecificCurrencyPair, TradePlace,
+};
 use crate::core::exchanges::events::{
     BalanceUpdateEvent, ExchangeBalance, ExchangeBalancesAndPositions, ExchangeEvent,
     LiquidationPriceEvent, Trade,
@@ -909,10 +909,14 @@ impl Exchange {
         //     DataRecorder.Save(liquidationPrice);
         // }
     }
+}
 
-    #[cfg(debug_assertions)]
-    pub fn get_specific_currency_pair(&self, currency_pair: CurrencyPair) -> SpecificCurrencyPair {
-        self.exchange_client
-            .get_specific_currency_pair(currency_pair)
-    }
+/// Helper method only for tests
+pub fn get_specific_currency_pair_for_tests(
+    exchange: &Exchange,
+    currency_pair: CurrencyPair,
+) -> SpecificCurrencyPair {
+    exchange
+        .exchange_client
+        .get_specific_currency_pair(currency_pair)
 }

--- a/exchanges/binance/tests/binance/binance_builder.rs
+++ b/exchanges/binance/tests/binance/binance_builder.rs
@@ -123,7 +123,7 @@ impl BinanceBuilder {
         }
 
         let default_price = get_default_price(
-            exchange.get_specific_currency_pair(OrderProxy::default_currency_pair()),
+            get_specific_currency_pair_for_tests(&exchange, OrderProxy::default_currency_pair()),
             &hosts,
             &settings.api_key,
         )

--- a/exchanges/binance/tests/control_panel/statistics.rs
+++ b/exchanges/binance/tests/control_panel/statistics.rs
@@ -35,6 +35,7 @@ use tokio::time::sleep;
 use crate::binance::common::get_default_price;
 use crate::get_binance_credentials_or_exit;
 use core_tests::order::OrderProxy;
+use mmb_core::core::exchanges::general::exchange::get_specific_currency_pair_for_tests;
 
 #[derive(Default, Clone, Debug, Deserialize, Serialize)]
 pub struct TestStrategySettings {}
@@ -131,7 +132,7 @@ async fn orders_cancelled() {
         Some("FromOrdersCancelledTest".to_owned()),
         CancellationToken::default(),
         get_default_price(
-            exchange.get_specific_currency_pair(test_currency_pair),
+            get_specific_currency_pair_for_tests(&exchange, test_currency_pair),
             &Binance::make_hosts(is_margin_trading),
             &api_key,
         )

